ament_cmake_core/cmake/environment_hooks/environment/path.sh

```diff
@@ -1,3 +1,5 @@
 # copied from ament_cmake_core/cmake/environment_hooks/environment/path.sh
 
-ament_prepend_unique_value PATH "$AMENT_CURRENT_PREFIX/bin"
+if [ -f "$AMENT_CURRENT_PREFIX/bin" ]; then
```

Comment:
Shouldn't this be a `-d` not `-f` is "file is a regular file (not a directory or device file)" https://www.tldp.org/LDP/abs/html/fto.html

Reply:
Good catch. Fixed in 68e1be993897cf96e235aafbc2d91578d52ba933.

:thinking: why it still had the expected ~20 directories in `PATH`...